docker-compose-dev.yml

```diff
@@ -12,7 +12,8 @@ services:
       DASH_DEBUG_MODE: "True"
       DASH_SILENCE_ROUTES_LOGGING: "False"
       DASH_SERVER_PORT: 8050
-      DB_HOST: db
+      # DB_HOST: db
+      DB_HOST: "mongodb://db/openpath_prod_durham"
```

Comment:
This should not be hardcoded since the DB will change depending on the dataset used for testing. Please revert.